src/UI/Root/Root.tsx

```diff
@@ -31,14 +30,14 @@ export const Root: React.FC = () => {
   const GlobalStyleProxy: any = GlobalStyles;
 
   return (
-    <QueryClientProvider client={queryClient}>
+    <>
       <ReactQueryDevtools initialIsOpen={false} />
       <GlobalStyleProxy />
       <KeycloakProvider>
```

Comment:
Since you have the new authProvider, `<KeycloakProvider>` can be removed.

Reply:
Yes that's the idea, but I don't want to remove everything at once as that would be too much to handle at once :D